certified-connectors/Map Pro/apiDefinition.swagger.json

```diff
@@ -0,0 +1,446 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Map Pro",
+    "description": "Map Pro is an application for Microsoft Teams that displays a map in any of your channels. This connector will allow you to manage existing places or create new ones.",
+    "termsOfService": "https://www.teams-pro.com/en/terms-of-use",
+    "version": "v1",
+    "contact": {
+      "name": "TeamsPro Support",
+      "url": "https://www.teams-pro.com/en/contact-us/",
+      "email": "teams-pro-help@witivio.com"
+    }
+  },
+  "host": "graph.teams-pro.com",
+  "basePath": "/map/v1/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/{mapId}/places/{placeId}": {
+      "get": {
+        "tags": [
+          "Places"
+        ],
+        "summary": "Get a place",
+        "description": "Get a place by it's unique identifier",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "mapId",
+            "description": "The unique identifier of the map",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Map ID",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "in": "path",
+            "name": "placeId",
+            "description": "The unique identifier of the place",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Place ID",
+            "x-ms-url-encoding": "single"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Return the place",
+            "schema": {
+              "$ref": "#/definitions/Place"
+            }
+          },
+          "400": {
+            "description": "The provided id is invalid"
+          },
+          "404": {
+            "description": "The place has not been found or does not exists"
```

Comment:
Please review all descriptions, titles, summaries, etc. They should be free from grammar and spelling errors. We recommend using spelling/grammar checking tools (for example exists -> exist).